### PR TITLE
chore(ui): update tools to support typescript

### PR DESCRIPTION
**Pull request by xla**
Most recently [typescript support](https://svelte.dev/blog/svelte-and-typescript) landed in Svelte. Along with it the consolidation of supporting tools. This change-set brings all these deps to the latest version and enables us to slowly transition across the codebase.

### CODE REVIEW

File: ui/DesignSystem/Component/Placeholder.svelte [hunk 1]
```diff
@@ -2,7 +2,7 @@
   export let style = null;
 </script>
 
-<style>
+<style type="text/css">
```
Comment (sarahscott) [inline ui/DesignSystem/Component/Placeholder.svelte:5]: why is this changed here and not elsewhere?

Comment (xla) [reply ui/DesignSystem/Component/Placeholder.svelte:5]: Oops, this is a debug leftover. Gonna remove.
